### PR TITLE
Lichess Support (should close issue #2)

### PR DESCRIPTION
**Pull request by hlee131**
This pull request should extend support to Lichess. However, instead of using `mutations.forEach`, it now uses the first element in `mutations`. Reasoning behind this was because the observer on Lichess was for some reason receiving three mutations every time, resulting in three different audios. This shouldn't break anything on chess.com or Lichess as tested locally, but if it does, let me know. 

Thanks.

### CODE REVIEW

File: js/chessMoveReminder.js [hunk 1]
```diff
@@ -27,71 +26,84 @@ function parseSecondsFromClock(clock) {
...
     let timer;
-    let seconds = 5;
     let audio;
 
-    let playAudio = async function(who) {
-        let audioUrl = 'audio/SoundEffect/SoundEffect.mp3';
-        if (who === 'HikaruGotham') {
-            audioUrl = 'audio/HikaruGotham/' + randomPositiveNumberWithoutRepeat(29) + '.wav';
+    let playAudio = async function (who) {
+        let audioUrl = "audio/SoundEffect/SoundEffect.mp3";
```
Comment (shuhaoli) [inline js/chessMoveReminder.js:32]: Please leave the quotes as single quotes: https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes

(Here and all future instances)

File: js/chessMoveReminder.js [hunk 2]
```diff
@@ -27,71 +26,84 @@ function parseSecondsFromClock(clock) {
...
+                );
+            }
         });
 
         observer.observe(target, {
-            attributes:true,
-            attributeFilter:['class']
+            attributes: true,
+            attributeFilter: ["class"],
+            attributeOldValue: true,
```
Comment (shuhaoli) [inline js/chessMoveReminder.js:100]: Is the old value being used somewhere?

**Review by shuhaoli — COMMENTED** [review]
Thanks for the work! I tested it on lichess, it seems to work in a normal game when it is set to a flat amount of time, however it doesn't seem to be working with % of remaining time. Also when spectating a game, a bunch of voices overlap for some reason, could you please take a look?

File: js/chessMoveReminder.js (hunk 2 above)
Comment (hlee131) [reply js/chessMoveReminder.js:100]: Sorry, was using it for debugging. I'll remove it.

**Comment by hlee131** [comment]
Thanks for the feedback! I will take a look.

**Comment by shuhaoli** [comment]
Did your new commits resolve both issues, or should I wait for more before testing?

**Comment by hlee131** [comment]
There are still some outstanding issues, such as the overlapping voices. That issues seems to be caused by multiple `MutationRecords` occurring even though only one attribute change is happening. Would you happen to have any idea why this is occurring ?

**Comment by hlee131** [comment]
I added in a manual comparison to test whether a class change occurred because during my testing, I found that the `MutationObserver` can be triggered in Lichess even when classes don't change. For example, it is triggered whenever I resized the window.

**Comment by shuhaoli** [comment]
Thanks for the work!

**Comment by shuhaoli** [comment]
Do you have a preferred name for me to shout you out with?

**Comment by hlee131** [comment]
Just my GitHub username will be fine.

**Comment by hlee131** [comment]
I just looked at the code after a `git fetch` and it looks like the code is different from what I wrote, mainly just one line. The observer should also have `attributeOldValue` set to true now that we do use it (line 102).

**Comment by shuhaoli** [comment]
Good catch, must've been a mistake I left in when updating something else. Thanks for noticing

**Pull request merged**